packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -20,13 +20,23 @@ import {
   buildObjEntity,
   buildParamEntity,
 } from './entity-builder';
+import {includesAtLeast} from './utils';
 
 export function resolveInterfaceMembers(
   entry: ApiEntryPoint,
-  apiInterface: ApiInterface
+  apiInterface: ApiInterface,
+  ancestorNames: string[]
 ): AnyEntity[] {
-  const members = resolveMembers(entry, apiInterface);
-  const inheritedMembers = resolveInheritedMembers(entry, apiInterface);
+  if (includesAtLeast(ancestorNames, apiInterface.name, 2)) {
```

Comment:
What if we used `.filter` here? We could simplify and remove the utils file and unit tests,

```
const matchingAncestors = ancestorNames.filter(name => name === apiInterface.name);

if (matchingAncestors.length >= 2) {
   return [];
}
```